packages/office-ui-fabric-react/src/components/pickers/BasePicker.tsx

```diff
@@ -112,7 +113,8 @@ export class BasePicker<T, P extends IBasePickerProps<T>> extends BaseComponent<
               aria-haspopup='true'
               autoCapitalize='off'
               autoComplete='off'
-              role='combobox' />
+              role='combobox'
+              disabled={ this.state.items.length >= itemLimit } />
```

Comment:
I see a couple of issues here:
1. The experience will be confusing as we don't provide any hint to user about why the control is disabled? IMO, we should have a hint text or message to do that.
2. Once I have reached the `itemLimit` there is no way to change the selection, a page refresh. No option for a case where I should add 4 items but i want to re-select some of them. page refresh is the only option then.